### PR TITLE
Feature/keywords

### PR DESCRIPTION
**Pull request by jamieadkins95**
Added a new field `keywords` that lists all of the relevant keywords for a card.

For example Clan Dimun Pirate Captain:

```
"152306": {
    ...
    "infoRaw": {
      "en-US": "<b><keyword=veteran>Veteran</keyword></b> 1.\nWhenever a Unit is <keyword=discard>Discarded</keyword> to your Graveyard, <keyword=boost>Boost</keyword> self by 1 while in your Hand, Deck or on your side of the Board.",
    },
    "keywords": [
      "veteran",
      "discard",
      "boost"
    ],
    "name": {
      "en-US": "Clan Dimun Pirate Captain",
    },
    ...
  },
```

### CODE REVIEW

File: master_xml.py [hunk 1]
```diff
@@ -295,6 +295,18 @@ def evaluateTokens(cards):
...
 
+def evaluateKeywords(cards):
+    for cardId in cards:
+        card = cards[cardId]
+        if card.get('infoRaw') == None:
+            continue
+        card['keywords'] = []
+        # Find all keywords in info string. E.g. find 'spawn' in '<keyword="spawn">'
+        # Can just use en-US here. It doesn't matter, all regions will return the same result.
+        result = re.findall(r'.*?\<keyword=(.*?)\>.*?', card['infoRaw']['en-US'])
```
Comment (Zolmeister) [inline master_xml.py]: not an issue, but the regex could be cleaner (no `.*?` or unnecessary escape `\<`)
```js
/<keyword=([^>]+)>/
```

**Comment by Zolmeister** [comment]
rebase, then g2g (squashed)

**Review by Zolmeister — APPROVED** [review]

File: master_xml.py (hunk 1 above)
Comment (jamieadkins95) [reply master_xml.py]: Fixed.

**Review by Tri125 — CHANGES_REQUESTED** [review]
Rebase on top of master, force push back to the Feature/keywords branch and it's good to go.

**Comment by jamieadkins95** [comment]
I'm honestly not a fan of rebasing. I'd prefer traceability of old branches over having a neat master branch.

Squashing will keep the master branch tidy, and I'd like to leave this branch as it is for full traceability, even if it is messy.

**Comment by Tri125** [comment]
Alright, no problem.

**Review by Tri125 — APPROVED** [review]

**Pull request merged**